test/server.js

```diff
@@ -188,4 +195,72 @@ describe("server", function() {
         done();
       });
   });
+
+  describe("when submitting only an address / chain pair", function(){
+
+    // Setup: write "Simple.sol" to repo
+    beforeEach(function(done){
+      chai.request(serverAddress)
+        .post('/')
+        .attach("files", read(simpleMetadataPath), "simple.meta.json")
+        .attach("files", read(simpleSourcePath), "Simple.sol")
+        .field("address", simpleInstance.options.address)
+        .field("chain", 'localhost')
+        .end(function (err, res) {
+          assert.equal(res.status, 200);
+          done();
+        });
+    });
+
+    afterEach(function(){
+      try { exec(`rm -rf ${mockRepo}`) } catch(err) { /*ignore*/ }
```

Comment:
Maybe just deleting created files?

Currently, when I have something in local repository folder everything gets deleted after running test.

Reply:
This should be fixed by a976d5b. 

Opted for configuring the repository name with an environment variable.

Reply:
Nice